client/client-multi/client-cpp/CMakeLists.txt

```diff
@@ -208,7 +207,7 @@ endif()
 # NOTE: Platform specific compilation flags should be set 
 # in the corresponding platform CMake script(s).
 if (NOT MSVC)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")
+    list(APPEND KAA_COMPILE_OPTIONS -fPIC)
```

Comment:
It works well both on 
gcc (Ubuntu 4.8.5-2ubuntu1~14.04.1) 4.8.5 
and
MSVC C/C++ Optimizing Compiler Version 19.00.23506 for x86